common/walker/templates/report.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Walker Report</title>
+    <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly.css">
```

Comment:
Sorry for overlooking this before. However, relying on CDNs like this, for a product, will cause some issues.

Assuming this is run in an air-gapped environment, this won't work. I am also not sure if this is compliant with privacy policies like GDPR.

A possible solution would be to provide an env-var (via clap) to customize this base URL. We could then add this as a static asset and configure it's location from the Helm charts.

Reply:
I don't see any fix for this. I see that there as an additional ConfigMap replacing this file, injecting this value via Helm. This seems overly complex to me and duplicates the while file.

I think having a way to replace the file is good. However, as we already have a template engine in place, and to replace elements in this file, we should simply re-use that same functionality for replacing the value as well. This would allow us to get rid of all the extra code below.

Reply:
I am not good at file operations in Rust. Rust can not do resource load like java, I think Rust just has two approach to get the file, compile it into Rust, or use absolute path. IMO, the code has two approach to it. one is to compile the file, I let it as a default, if the user gives an env param, we will use an absolute path and can use vaules.yaml for replace the URL.

Reply:
My method is too complicated. I will change it.

Reply:
To me it looks like there's still the link `unpkg`.